### PR TITLE
Add handling routes to BlackDuck and Alert Controllers

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/854#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: controllers/alert_mutator.go [hunk 1]
```diff
@@ -91,7 +102,7 @@ func (p *AlertPatcher) patchEnvirons() error {
 	for _, e := range p.alertCr.Spec.Environs {
 		vals := strings.Split(e, ":") // TODO - doesn't handle multiple colons
```
Comment (msenmurugan) [inline controllers/alert_mutator.go]: can you pls fix the multiple colons by using SplitN

File: controllers/blackduck_mutator.go [hunk 2]
```diff
@@ -149,24 +155,33 @@ func (p *BlackduckPatcher) patchSealKey() error {
 
 // TODO: common with Alert
 func (p *BlackduckPatcher) patchExposeService() error {
+
 	// TODO use contansts
-	id := fmt.Sprintf("Service.%s-blackduck-webserver-exposed", p.blackDuckCr.Name)
-	runtimeObject, ok := p.mapOfUniqueIdToBaseRuntimeObject[id]
+	routeID := fmt.Sprintf("Route.%s-blackduck-webserver-exposed", p.blackDuckCr.Name)
```
Comment (msenmurugan) [inline controllers/blackduck_mutator.go]: please verify the route name of what we kept for 2019.8.x release

**Comment by operator-r2d2** [comment]
@mphammer: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: controllers/blackduck_mutator.go (hunk 2 above)
Comment (mphammer) [reply controllers/blackduck_mutator.go]: Nice catch

**Pull request merged**